marimo/_output/formatters/structures.py

```diff
@@ -13,7 +13,10 @@
 from marimo._utils.flatten import CyclicStructureError, flatten
 
 
-def _leaf_formatter(value: object) -> bool | None | str | int:
+def _leaf_formatter(
```

Comment:
nice, thanks for updating. 

there might be tests somewhere for this. can we add some tests when `json_compat_values=True`

Reply:
sure, I'm waiting for latest approving of approaches before test writing

Reply:
ah yea makes sense. the approach looks great